### PR TITLE
Added badges to README.md

### PR DESCRIPTION
**Pull request by DerMoehre**

### CODE REVIEW

**Review by JoFrhwld — CHANGES_REQUESTED** [review]
If you could roll back this commit, and start working on the mean_stdv in a separate branch in your own repo (I’d recommend a branch called `feature/mean-np`), that would keep this particular PR cleaner.

**Comment by JoFrhwld** [comment]
All checks are failing, but I think that must be an issue with the GitHub CI, since it’s complaining on the import of numpy that there’s no such module.

This PR looks bigger than it is, I think because of the issue I introduced at some point where there’s duplicated commits with different hashes. 

But, there’s a also at least two different things being contributed here, I think: 

1. Readme Badges
2. Tests for the mean_stdv function 

It’ll keep the PRs cleaner if separate “features” like this were committed to separate branches in your own repo, and then you can send the pull request do dev from that feature branch, rather than from your master branch. Here, if you just roll back the final commit here where you introduce the numpy tests, that’ll clean things up for this specific PR.

File: README.md [hunk 1]
```diff
@@ -24,6 +24,8 @@ There may be a delay between when a bug is reported and when a bug is resolved.
 
 ## Attribution
 [![DOI](https://zenodo.org/badge/doi/10.5281/zenodo.22281.svg)](http://dx.doi.org/10.5281/zenodo.22281)
```
Comment (JoFrhwld) [inline README.md:26]: While you’re sprucing up the shields, could you replace this zenodo shield with this:

`[DOI](https://zenodo.org/badge/latestdoi/13744621)`

The current shield points at a particular DOI (which I think was all that was possible when I first set up the DOI minting). But the link pasted here will point to he most current DOI, which is the behavior we want.

Comment (JoFrhwld) [inline README.md:26]: We’re also on PyPi! 

`[PyPI version fury.io](https://pypi.python.org/pypi/fave/)`
[PyPI version fury.io](https://pypi.python.org/pypi/fave/)

**Review by JoFrhwld — CHANGES_REQUESTED** [review]

**Comment by chrisbrickhouse** [comment]
> If you could roll back this commit, and start working on the mean_stdv in a separate branch in your own repo (I’d recommend a branch called `feature/mean-np`), that would keep this particular PR cleaner.

You can do this with:
* `git checkout -b feature/mean-np` which will create a new branch with all these changes.
* `git push --set-upstream origin feature/mean-np` which will add that branch to GitHub. You can then create a new pull request from that branch on the GitHub website
* Now, go back to your master branch: `git checkout master`
* `git revert 5983ed6` then commit and push the result. This will remove the testing files from this pull request making the review easier.

**Comment by DerMoehre** [comment]
Thanks for the explanation. I now understand the usage of branches better :) 
Sorry for the mess

I will do this tomorrow morning before Work :)

Btw I follow you in LinkedIn ;-) maybe I can learn Something about this topic also there

**Comment by chrisbrickhouse** [comment]
> All checks are failing, but I think that must be an issue with the GitHub CI, since it’s complaining on the import of numpy that there’s no such module.

@JoFrhwld Yeah, apparently we don't install the package dependencies, just flake8 and pytest. Our set-up action can do this, see [v3 documentation](https://github.com/actions/setup-python/tree/98f2ad02fd48d057ee3b4d4f66525b231c3e52b6#caching-packages-dependencies), so I'll add that in before this gets merged.

**Comment by DerMoehre** [comment]
I reverted the PR. I will add those requested changes later

**Review by JoFrhwld — APPROVED** [review]

**Review by JoFrhwld — APPROVED** [review]

**Comment by DerMoehre** [comment]
I added the changes @JoFrhwld 
I am not sure, how to deal with this citation

`As of v1.1.3 onwards, releases from this repository will have a DOI associated with them through Zenodo. The DOI for the current release is [10.5281/zenodo.22281](http://dx.doi.org/10.5281/zenodo.22281). We would recommend the citation:`

Is it the same version of DOI you mentioned? If yes, I will add it in another PR

**Pull request merged**